### ISSUE TITLE
Wrong path in case of relativeToChangelogFile = true and databaseChangeLog has logicalFilePath != PhisicalFilePath

### ISSUE THREAD

**Issue by kezhevatov** [liquibase/liquibase#1353]
https://github.com/liquibase/liquibase/blob/99e76caad1c5c0060a030e48853f163c47a4f383/liquibase-core/src/main/java/liquibase/change/core/SQLFileChange.java#L121

Can't find file in case of relativeToChangelogFile = true and databaseChangeLog has logicalFilePath attribute != PhisicalFilePath
In this case getChangeSet().getFilePath() returns logicalFilePath and 
inputStream = Scope.getCurrentScope().getResourceAccessor().openStream(relativeTo, path); 
can't find this path.

At previous version work correct cause path equals:
base = changeSet.getChangeLog().getPhysicalFilePath().replaceAll("\\\\","/");

**Comment by molivasdat** [comment]
Hi @kezhevatov Thanks for writing up this issue. Which version were you using previously?  Which version are you using now that shows the error?

**Comment by kezhevatov** [comment]
Version with error is 4.0.0. Previously version is 3.8.7, example of code from liquibase.util.StreamUtil.openFromClasspath(...

**Comment by QuadStingray** [comment]
Issue is still active in liquibase 4.2.0

```
<databaseChangeLog
        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.2.xsd"
        logicalFilePath="filePath.xml"
>

    <changeSet id="insert csv values to Table" >
        <preConditions onFail="MARK_RAN">
            <rowCount tableName="TABLE_NAME" expectedRows="0"/>
        </preConditions>
        <loadData tableName="TABLE_NAME" file="data.csv" relativeToChangelogFile="true"/>
    </changeSet>

</databaseChangeLog>

**Comment by hackmann** [comment]
Is this fixed for `loadData`? The linked pull request is only about `sqlFile`

**Comment by hackmann** [comment]
The problem I have with `loadData` and usage of `relativeToChangelogFile="true"` is still present in 4.4.3 and 4.6.2 and finally fixed in #2073 (Fix load relative file). Which is merged but not yet included in a released version.

**Comment by molivasdat** [comment]
@hackmann Yes, #2073 should be included in v4.6.3 which should be released soon.
